pkg/kubelet/dockershim/docker_stats_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !dockerless
+// +build linux,!dockerless windows,!dockerless
```

Comment:
Wondering why we want to include linux, and !dockerless windows tag for docker_stats_test.go ? in docker_stats.go, it only have !dockerless tag. 

From the code base, didn't see `!dockerless windows` was used anywhere before, is it something new or you are referring to windows ?

Reply:
If `linux,!dockerless` is enough, it would be great.

My purpose is that TestContainerStats is not supported on Mac.

Updated. Remove `windows,!dockerless`.

Reply:
If `linux,!dockerless` is enough, it would be great.

My purpose is that TestContainerStats is not supported on Mac.

Updated. Remove `windows,!dockerless`.